### ISSUE TITLE
Upgrade 1.3.10 -> 1.4.6 UsersModule

### ISSUE THREAD

**Issue by Kaik** [zikula/core#3797]
| Q                 | A
| ----------------- | ---
| Zikula Version    | 1.4.6
| PHP Version       | 7..?

### Expected behavior
No error.

### Actual behavior
Error:
```
Module "Users" does not exist or it is not enabled.

500 Internal Server Error - InvalidArgumentException

Stack Trace

in lib/Zikula/Bundle/CoreBundle/HttpKernel/ZikulaKernel.php at line 180   -
    public function getModule($moduleName, $first = true)
    {
        if (!isset($this->moduleMap[$moduleName])) {
            throw new \InvalidArgumentException(sprintf('Module "%s" does not exist or it is not enabled.', $moduleName, get_class($this)));
        }
        if (true === $first) {
at ZikulaKernel ->getModule ('Users') 
in system/ExtensionsModule/Helper/ExtensionHelper.php at line 122   -
        if ($extension->getType() == self::TYPE_SYSTEM) {
            // system modules are always loaded
            $bundle = $this->container->get('kernel')->getModule($extension->getName());
        } else {
            $bundle = $this->forceLoadExtension($extension);
        }
at ExtensionHelper ->upgrade (object(ExtensionEntity)) 
in lib/Zikula/Bundle/CoreInstallerBundle/Stage/Upgrade/InitStage.php at line 139   -
    private function upgradeUsersModule()
    {
        $usersModuleEntity = $this->container->get('zikula_extensions_module.extension_repository')->findOneBy(['name' => 'ZikulaUsersModule']);
        $this->container->get('zikula_extensions_module.extension_helper')->upgrade($usersModuleEntity);
    }
}
at InitStage ->upgradeUsersModule () 
in lib/Zikula/Bundle/CoreInstallerBundle/Stage/Upgrade/InitStage.php at line 49   -
            return false;
        }
        $this->init();
        $this->upgradeUsersModule();
        return false;
    }
at InitStage ->isNecessary () 
in vendor/zikula/wizard/Wizard.php at line 106   +
at Wizard ->getCurrentStage ('init') 
in lib/Zikula/Bundle/CoreInstallerBundle/Controller/UpgraderController.php at line 57   -
        // begin the wizard
        $wizard = new Wizard($this->container, realpath(__DIR__ . '/../Resources/config/upgrade_stages.yml'));
        $currentStage = $wizard->getCurrentStage($stage);
        if ($currentStage instanceof WizardCompleteInterface) {
            return $currentStage->getResponse($request);
        }
at UpgraderController ->upgradeAction (object(Zikula_Request_Http), 'init')
at call_user_func_array (array(object(UpgraderController), 'upgradeAction'), array(object(Zikula_Request_Http), 'init')) 
in vendor/symfony/symfony/src/Symfony/Component/HttpKernel/HttpKernel.php at line 144   +
at HttpKernel ->handleRaw (object(Zikula_Request_Http), '1') 
in vendor/symfony/symfony/src/Symfony/Component/HttpKernel/HttpKernel.php at line 64   +
at HttpKernel ->handle (object(Zikula_Request_Http), '1', true) 
in vendor/symfony/symfony/src/Symfony/Component/HttpKernel/DependencyInjection/ContainerAwareHttpKernel.php at line 69   +
at ContainerAwareHttpKernel ->handle (object(Zikula_Request_Http), '1', true) 
in vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php at line 185   +
at Kernel ->handle (object(Zikula_Request_Http)) 
in index.php at line 23   +

```

### Steps to reproduce
I'm using friends db which is 1.3.10 before start users module is named Users while after error users module name is ZikulaUsersModule so it might be a bug.

**Comment by Kaik** [comment]
Ok I was able to go pass this point by adding ```        $em = $this->container->get('doctrine')->getEntityManager();
        $em->clear();```
Zikula\Bundle\CoreInstallerBundle\Stage\Upgrade\InitStage
I still had one error before regarding bootstrap path for the theme.. I will add this error as well

**Comment by craigh** [comment]
THERE IS NO 1.4.7 and there never will be. please test actual releases.

**Comment by Kaik** [comment]
by 1.4.7 I meant current 1.4 branch which it is not even with 1.4.6 tag and some issues I had are fixed there already...

**Comment by craigh** [comment]
anything that is fixed there is also in the 1.5 branch. the 1.4 branch is dead.

in any case, there is no sense testing upgrades with developmental code. Use released software please. In this case you should be testing with [1.5.1](https://github.com/zikula/core/releases/tag/1.5.1)

**Comment by Kaik** [comment]
Well, there is only one reason why I have pursuit upgrade on 1.4.6 - I just had it done before and my production DB is "corrupted" I need to do fixes before upgrade duplicate entries etc.. I just wanted to replicate previous success and move to 1.5.0 now 1.5.1 
I have just upgraded 1.3.12 to core 1.4.6 I will test that other DB as well and will move to 1.5.1 and authentication error. Since 1.4 is dead this ticket is not needed.
